src/main/scala/com/workday/warp/common/spec/WarpJUnitSpec.scala

```diff
@@ -3,7 +3,7 @@ package com.workday.warp.common.spec
 import com.workday.telemetron.spec.HasTelemetron
 import com.workday.warp.common.utils.TryMatchers
 import org.scalatest._
-import org.scalatest.junit.{AssertionsForJUnit, JUnitSuite}
+import org.scalatestplus.junit.{AssertionsForJUnit, JUnitSuite}
```

Comment:
these types in `scalatest` package are deprecated type aliases that forward to the real impls in package `scalatestplus`